appeals/web/src/server/lib/mappers/appellantCase.mapper.js

```diff
@@ -29,348 +31,151 @@ export function initialiseAndMapData(appellantCaseData, appealDetails, currentRo
 	currentRoute =
 		currentRoute[currentRoute.length - 1] === '/' ? currentRoute.slice(0, -1) : currentRoute;
 
+	/**
+	 * Generates a document field with defaults
+	 *
+	 * @param {Object} options
+	 * @param {string} options.id
+	 * @param {string} options.text
+	 * @param {FolderInfo|null|undefined} options.folderInfo
+	 * @param {string} [options.cypressDataName]
+	 * @returns {Instructions}
+	 */
+	const documentField = ({ id, text, folderInfo, cypressDataName }) => {
```

Comment:
This is defined here so it can access the session, appeal ID, etc... - these parts are common for each document row, so its just a small wrapper to set those values. It can't be in the top scope as then it wouldn't have access to those values.

Reply:
I think this would be a bit clearer and more consistent if we just passed those values as parameters, then it wouldn't be dependent on the scope